packages/client/src/utils/errors.tsx

```diff
@@ -8,6 +8,23 @@ import { tryFindMessageDescriptor } from "@gc-digital-talent/i18n";
 export const extractErrorMessages = (combinedError: CombinedError) =>
   combinedError.graphQLErrors.flatMap((error) => error.message);
 
+// grab the validation error enums to map them to messages, guarding against possible missing fields
+export const extractValidationMessageKeys = (
+  combinedError: CombinedError,
+): string[] | null => {
+  const { extensions } = combinedError.graphQLErrors[0];
```

Comment:
It makes me nervous to access this array without first checking the length.  Also, what if there are more than one error?  Maybe you could use a flatMap like the function above so it works regardless of the length?

Reply:
Attempted in https://github.com/GCTC-NTGC/gc-digital-talent/pull/7596/commits/a90863430b959908199ce039635c4ffd8a0ac476